lib/widgets/content.dart

```diff
@@ -167,6 +169,22 @@ class Heading extends StatelessWidget {
   }
 }
 
+class ThematicBreak extends StatelessWidget {
```

Comment:
nit: let's order these classes the same way as their invocations appear in `BlockContentList`  above.

(Then similarly for the test code.)